Gemfile

```diff
@@ -6,7 +6,7 @@ gem 'rails', '~>6.1.0'
 
 group :development, :test do
   gem 'byebug'
-  gem 'factory_bot_rails'
+  gem 'factory_bot_rails', require: false
```

Comment:
Whats the reason for adding this require flag? (Just curious as im not sure what it does)